desc/coils.py

```diff
@@ -2878,3 +2879,127 @@ def _linking_number(x1, x2, x1_s, x2_s, dx1, dx2):
     small = dx_norm < jnp.finfo(x1.dtype).eps
     ratio = jnp.where(small, 0.0, num / jnp.where(small, 1.0, den))
     return ratio.sum()
+
+
+def initialize_modular_coils(eq, num_coils, r_over_a=1.5):
+    """Initilize a CoilSet of modular coils for stage 2 optimization.
+
+    The coils will be centered on the equilibrium magnetic axis, and aligned such that
+    the normal to the coil points along the axis. The currents will be set to match
+    the equilibrium required poloidal linking current.
+
+    The coils will be FourierPlanarCoils, if another type is desired use
+    ``coilset.to_FourierXYZ()``, ``coilset.to_SplineXYZ()`` etc.
+
```

Comment:
Possibly add a note about checking that they do not intersect the plasma? though this is a common sense thing to check before optimization I guess